tableauserverclient/models/tableau_types.py

```diff
@@ -27,5 +29,9 @@ class Resource:
 def plural_type(content_type: Resource) -> str:
     if content_type == Resource.Lens:
         return "lenses"
+    elif content_type == Resource.Datarole:
```

Comment:
this isn't necessary: lenses is a special case because it can't just add "s" on the end, but datarole and metric can use the default behavior.

Reply:
Understood. So should I delete this part and create new PR? Not sure why this PR was closed. 
Thanks

Reply:
No, that was an accident, sorry. You can just make the update in this PR.

Reply:
Ok, np, updated